tox.ini

```diff
@@ -11,6 +11,8 @@ basepython =
     py38: python3.8
     py39: python3.9
     py310: python3.10
+    py311: python3.11
```

Comment:
3.8 mentioned above?

Reply:
@seanpearsonuk, this is just for nomenclature.
basepython =
    py37: python3.7
    py38: python3.8
    py39: python3.9
    py310: python3.10
    py311: python3.11
    py312: python3.12
    py: python3


Shall we remove the py37 and py38?